src/components/Login.jsx

```diff
@@ -1,39 +1,53 @@
-import React from 'react';
+import { useState } from 'react';
+import axios from 'axios';
 import { Link as RouterLink } from 'react-router-dom';
+import { BACKEND_LOGIN_URL } from '../constants/constants';
 import { LockOutlined, UserOutlined } from '@ant-design/icons';
-import { Button, Checkbox, Form, Input } from 'antd';
+import { Button, Form, Input } from 'antd';
+import { HttpStatusCode } from 'axios';
 const Login = () => {
-  const onFinish = (values) => {
-    console.log('Received values of form: ', values);
+  const [name, setName] = useState('');
+  const [password, setPassword] = useState('');
+  const handleSubmit = async (e) => {
+    try {
+      e.preventDefault();
+      const response = await axios.post(BACKEND_LOGIN_URL, { name, password });
+      const token = response.data;
+      localStorage.setItem('accessToken', token.accessToken);
+      localStorage.setItem('refreshToken', token.refreshToken);
+      alert('Login successful');
+      setName('');
+      setPassword('');
+      window.location.href = '/StartPage';
+    } catch (error) {
```

Comment:
If this is for page navigation you should use `useNavigate` coming from react router